src/pages/HomePage/index.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components'
+import AddMovieButton from './AddMovieButton'
+import GenreFilters from './GenreFilters'
+
+import NetflixLogo from '../../common/NetflixLogo'
+import SearchBox from './SearchBox'
+import SearchResults from './SearchResults'
+import SortDropdown from './SortDropdown'
+import Footer from './Footer'
+
+const Wrapper = styled.div`
+  .header {
+    height: 396;
+    background-image: url(/assets/header.png);
```

Comment:
it's possbile to import image directly? then that image is involved into the management of webpack, so that webpack is able to optimize images based on some particular plugins  for production env.